mkdocs/docs/reference/tasks/common.md

```diff
@@ -61,20 +115,8 @@ In the [example above](#example), a YAML file corresponding to the baseline or c
 
 If this task exits with a non-zero error code, the experiment to which it belongs will fail.
 
-### Dynamic placeholder substitution
-
-Inputs to tasks can contain placeholders, or template variables, which will be dynamically substituted when the task is executed by Iter8. In the [example above](#example), one input is:
-```shell
-kubectl apply -f https://raw.githubusercontent.com/iter8-tools/iter8/master/samples/knative/quickstart/{{ .promote }}.yaml
-```
-In this case, the placeholder is `{{ .promote }}`. 
-
-Placeholder substitution in task inputs works as follows. 
-
-Iter8 will find the version recommended for promotion which is determined by Iter8 during the  course of the experiment, and stored in the `status.versionRecommendedForPromotion` field of the experiment resource. The version recommended for promotion is the winner, if a winner has been found in the experiment. Otherwise, it is the baseline version supplied in the `spec.versionInfo` field of the experiment.
-
-If the placeholder is `{{ .name }}`, Iter8 will substitute it with the name of the version recommended for promotion. If it is any other variable, Iter8 will substitute it with the value of the corresponding variable for the version recommended for promotion. Variable values are specified in the variables field of the version detail when the experiment is created.
+### Limitations
```

Comment:
The deprecation is fine. However, I think the "limitations" title is incorrect. `disableInterpolation` flag is a feature, not a limitation for instance :) IMO, the only limitation of `exec` over `bash` is its verbosity. `bash` will cut out a line or two in many situations, which is certainly a very welcome type of simplicity.

Also, when there is no version recommended for promotion, but interpolation is required, the two tasks behave differently (in the case of `exec`, it will fail, and in the case of `bash` it will quietly output something which is probably not the desirable output anyway).